cmd/shell.go

```diff
@@ -80,6 +82,32 @@ func (cl cfsslLogger) Emerg(msg string) {
 	cl.AuditErr(msg)
 }
 
+type grpcLogger struct {
+	blog.Logger
+}
+
+func (log grpcLogger) Fatal(args ...interface{}) {
+	log.Print(args...)
+	os.Exit(1)
+}
+func (log grpcLogger) Fatalf(format string, args ...interface{}) {
+	log.Printf(format, args...)
+	os.Exit(1)
+}
+func (log grpcLogger) Fatalln(args ...interface{}) {
+	log.Println(args...)
+	os.Exit(1)
+}
+func (log grpcLogger) Print(args ...interface{}) {
+	log.AuditErr(fmt.Sprint(args...))
+}
+func (log grpcLogger) Printf(format string, args ...interface{}) {
+	log.AuditErr(fmt.Sprintf(format, args...))
+}
+func (log grpcLogger) Println(args ...interface{}) {
+	log.AuditErr(fmt.Sprintln(args...))
+}
+
 // StatsAndLogging constructs a Statter and an AuditLogger based on its config
 // parameters, and return them both. Crashes if any setup fails.
 // Also sets the constructed AuditLogger as the default logger.
```

Comment:
Comment should probably be extended to note it should only be called before any other gRPC stuff is done.